cmd/inspect.go

```diff
@@ -23,9 +24,11 @@ import (
 )
 
 const maxTableFieldLen = 50
+const PageWidth = 80
```

Comment:
[nit] Needs to be exported?
```suggestion
const pageWidth = 80
```